docs/standard/architecture/toc.md

```diff
@@ -0,0 +1,77 @@
+### [Containerizing monolithic applications](architecting-container-and-microservice-based-applications/containerizing-monolithic-applications.md)
+### [State and data in Docker applications](architecting-container-and-microservice-based-applications/state-and-data-in-docker-applications.md)
+### [Service-oriented architecture ](architecting-container-and-microservice-based-applications/service-oriented-architecture-.md)
+### [Microservices architecture](architecting-container-and-microservice-based-applications/microservices-architecture.md)
+### [Data sovereignty per microservice](architecting-container-and-microservice-based-applications/data-sovereignty-per-microservice.md)
+### [Logical architecture versus physical architecture](architecting-container-and-microservice-based-applications/logical-architecture-versus-physical-architecture.md)
+### [Challenges and solutions for distributed data management](architecting-container-and-microservice-based-applications/challenges-and-solutions-for-distributed-data-management.md)
+### [Identifying domain-model boundaries for each microservice](architecting-container-and-microservice-based-applications/identifying-domain-model-boundaries-for-each-microservice.md)
+### [Communication between microservices](architecting-container-and-microservice-based-applications/communication-between-microservices.md)
+### [Asynchronous message-based communication](architecting-container-and-microservice-based-applications/asynchronous-message-based-communication.md)
+### [Creating, evolving, and versioning microservice APIs and contracts](architecting-container-and-microservice-based-applications/creating,-evolving,-and-versioning-microservice-apis-and-contracts.md)
+### [Microservices addressability and the service registry](architecting-container-and-microservice-based-applications/microservices-addressability-and-the-service-registry.md)
+### [Creating composite UI based on microservices, including visual UI shape and layout generated by multiple microservices](architecting-container-and-microservice-based-applications/creating-composite-ui-based-on-microservices,-including-visual-ui-shape-and-layout-generated-by-multiple-microservices.md)
+### [Resiliency and high availability in microservices](architecting-container-and-microservice-based-applications/resiliency-and-high-availability-in-microservices.md)
+### [Orchestrating microservices and multi-container applications for high scalability and availability](architecting-container-and-microservice-based-applications/orchestrating-microservices-and-multi-container-applications-for-high-scalability-and-availability.md)
+### [Using Azure Service Fabric](architecting-container-and-microservice-based-applications/using-azure-service-fabric.md)
+## [Development Process for Docker Based Applications](development-process-for-docker-based-applications/)
```

Comment:
this toc node looks weird. I can access the sub-node. Perhaps if you add the index.md filename here, it fixes?